### ISSUE TITLE
Remove text shadow

### ISSUE THREAD

**Issue by MicrocontrollersDev** [mrbuilder1961/WheresMyChatHistory#109]
**Is your feature request related to a problem? Please describe.**

By default, the chat box already has a background, so text shadow does not necessarily help with readability. It would be nice to see an option to remove it.

**Describe the solution you'd like**

One or two toggles to disable text shadow. If two, then one could be for the normal chat hud, and the other could be for the chat input box.

**Describe alternatives you've considered**

None

**Additional context**

None

**Comment by mrbuilder1961** [comment]
yeah this should be pretty easy! should i add an option for the search field as well? do note that if it gets a config opt too, it wouldn't affect the placeholder text (Search...)

**Comment by MicrocontrollersDev** [comment]
I think that would be nice yeah

**Comment by mrbuilder1961** [comment]
ok i'm so sorry im taking forever on this. i thought this would be easy so i kept putting it off and now that im finally getting around to it, mojang completely changed how rendering works, which makes implementing this on newer versions a huge pain in the ass.

it's coming though. thanks for your patience 🫡

**Comment by mrbuilder1961** [comment]
still working on it, still fighting mojangs demons they created in the newer versions to do their text-render-bidding. i'll push the partial feature (working on <=1.21.11) tomorrow and i'll try to wrap up the new impl but if push comes to shove i'll have to delay it until another release.

again thanks for your immense patience 🙏 🙏 🙏
